lib/ravelin/client.rb

```diff
@@ -118,5 +119,31 @@ def faraday_options
         }
       }
     end
+
+    def faraday_proxy_options
+      {
+        request: { timeout: Ravelin.faraday_timeout },
+        headers: {
+          'Content-Type'  => 'application/json; charset=utf-8'.freeze,
+          'User-Agent'    => "Ravelin Proxy RubyGem/#{Ravelin::VERSION}".freeze
+        }
+      }
+    end
+
+  end
+
+  class ProxyClient < Client
```

Comment:
I'm wondering if we should move this to its own file?

Reply:
fixed in: 07482e17f94355d4943572a957e7bf9e3380f46d